elena/domain/services/generic_bot.py

```diff
@@ -112,6 +112,12 @@ def limit_min_amount(self) -> Optional[float]:
             self._logger.error(f"Error getting limit min amount: {err}", error=err)
             return None
 
+    def amount_to_precision(self, amount: float) -> float:
+        return self.manager.amount_to_precision(self.exchange, self.pair, amount)
+
+    def price_to_precision(self, price: float) -> float:
+        return self.manager.price_to_precision(self.exchange, self.pair, price)
+
```

Comment:
wont we call `amount_to_precision` & `price_to_precision` before calling exchange, in exchange manager?